xmtp_mls/src/groups/tests/test_commit_log_remote.rs

```diff
@@ -387,7 +388,7 @@ async fn test_download_commit_log_from_remote() {
     // We should have published 4 commits
     assert_eq!(
         test_results[0].publish_commit_log_results.clone().unwrap()[0].num_entries_published,
-        4
+        5
```

Comment:
Why did this change?

Reply:
looks like `send_message` fires `maybe_insert_key_update_intent`

https://github.com/xmtp/libxmtp/blob/5affa3eb9a877e88158205c8a1bdf4c0214c4528/xmtp_mls/src/groups/intents/queue.rs#L158